src/pages/HomePage/HomePage.style.js

```diff
@@ -1,13 +1,13 @@
 import styled from 'styled-components';
 
 const Heading1 = styled.h1`
```

Comment:
I would create global styles that define the headlines. Those are usually reused throughout an app and shouldn't be defined specifically inside the homepage component